pkg/app/web/src/components/deployments-page/deployment-filter/index.tsx

```diff
@@ -78,9 +78,25 @@ export const DeploymentFilter: FC<DeploymentFilterProps> = memo(
       }
     }, [applications, options]);
 
-    const [labelOptions, setLabelOptions] = useState(new Array<string>());
+    const [allLabels, setAllLabels] = useState(new Array<string>());
     const [selectedLabels, setSelectedLabels] = useState(new Array<string>());
 
+    useEffect(() => {
+      const seen = new Map<string, true>();
+      const labels = new Array<string>();
```

Comment:
nits, we can use Typescript Set and add labels to Set which ensures no duplication, then use `Array.from(set)` to make labels array after.

Reply:
That's what I was looking for!